package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@httptoolkit/browser-launcher",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
You don't need to manually bump the version in PRs. That's done automatically later when a release is made (via `npm version major/minor/patch`). It doesn't need to be done manually, and doing so will mess with npm's automatic bumping & tagging logic (in practice, it means we'll skip version 2.0.1 entirely).